awesomeConfig.cmake

```diff
@@ -355,6 +360,33 @@ foreach(file ${AWESOME_CONFIGURE_FILES})
                    ESCAPE_QUOTES
                    @ONLY)
 endforeach()
+
+set(AWESOME_CONFIGURE_COPYONLY_WITHCOV_FILES
+    ${awesome_c_configure_files}
+    ${awesome_lua_configure_files}
+)
+
+if(DO_COVERAGE)
+    foreach(file ${AWESOME_CONFIGURE_COPYONLY_WITHCOV_FILES})
+        configure_file(${SOURCE_DIR}/${file}
+                    ${BUILD_DIR}/${file}
+                    COPYONLY)
+    endforeach()
+
+    # There is no way to avoid that one
+    configure_file(${SOURCE_DIR}/lib/awful/util.lua
+                    ${BUILD_DIR}/lib/awful/util.lua
+                    ESCAPE_QUOTES
+                    @ONLY)
+else()
+    foreach(file ${AWESOME_CONFIGURE_COPYONLY_WITHCOV_FILES})
+        configure_file(${SOURCE_DIR}/${file}
+                    ${BUILD_DIR}/${file}
+                    ESCAPE_QUOTES
+                    @ONLY)
+    endforeach()
+endif()
+
```

Comment:
Sigh. More bad hacks and band-aids. My long-term plan is still to change things so that bug reports with line numbers are useful again, but for now if this hack makes you happy...